superchain/superchain_test.go

```diff
@@ -301,3 +301,25 @@ func TestCanyonTimestampOnBlockBoundary(t *testing.T) {
 		}
 	}
 }
+
+// TestAveoForkTimestamps ensures that network upgades that occur on a block boundary
+// also occur on Aveo which has a non-standard block time.
+func TestAveoForkTimestamps(t *testing.T) {
+	aevoGenesisL2Time := uint64(1679193011)
+	aevoBlockTime := uint64(10)
+	config := Superchains["mainnet"]
+	t.Run("canyon", testNetworkUpgradeTimestampOffset(aevoGenesisL2Time, aevoBlockTime, config.Config.CanyonTime))
+	t.Run("ecotone", testNetworkUpgradeTimestampOffset(aevoGenesisL2Time, aevoBlockTime, config.Config.EcotoneTime))
+}
```

Comment:
```suggestion
	aveoGenesisL2Time := uint64(1679193011)
	aveoBlockTime := uint64(10)
	config := Superchains["mainnet"]
	t.Run("canyon", testNetworkUpgradeTimestampOffset(aveoGenesisL2Time, aveoBlockTime, config.Config.CanyonTime))
	t.Run("ecotone", testNetworkUpgradeTimestampOffset(aveoGenesisL2Time, aveoBlockTime, config.Config.EcotoneTime))
}
```
typo `aevo`/`aveo`

Reply:
It's actually aevo. I used aveo incorrectly everywhere else. TY for finding.